examples/shardingsphere-proxy-example/pom.xml

```diff
@@ -33,5 +33,38 @@
         <module>shardingsphere-proxy-boot-mybatis-example</module>
         <module>shardingsphere-proxy-hint-example</module>
     </modules>
+
+    <build>
+		<pluginManagement>
```

Comment:
Please fix the indent here and other.

Reply:
In ide it looks correctly indented, I am not sure why it looks like here this way, I will correct the individual files indentation if it still happens.

Reply:
@TeslaCN The CI test are failing with error as they can't find the version of parent although I have added it correctly, and there are no dependencies in those pom files, I cannot understand what is missing, please help.

Reply:
Your indent are `tab`, replace them with `space`.